power/ReadVcdActivities.cc

```diff
@@ -175,13 +176,41 @@ ReadVcdActivities::setVarActivity(VcdVar *var,
   }
 }
 
+static std::string replaceAll(std::string_view str, std::string_view from,
+                                    std::string_view to) {
+  size_t start_pos = 0;
+  std::string result(str);
+  while ((start_pos = result.find(from, start_pos)) != std::string::npos) {
+    result.replace(start_pos, from.length(), to);
+    start_pos += to.length();
+  }
+  return result;
+}
+
 void
 ReadVcdActivities::setVarActivity(const char *pin_name,
                                   const VcdValues &var_values,
                                   int value_bit)
 {
-  const Pin *pin = sdc_network_->findPin(pin_name);
-  if (pin) {
+  std::vector<const Pin*> pins;
+  std::string pinname = replaceAll(pin_name, "\\", "");
```

Comment:
Cherry has a function that does something similar to this already in the string utilities I believe

Reply:
None that I can find